presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestStringStatisticsBuilder.java

```diff
@@ -315,7 +315,7 @@ private void assertMergedStringStatistics(List<ColumnStatistics> statisticsList,
 
     private static void assertMinMaxValuesWithLimit(Slice expectedMin, Slice expectedMax, List<Slice> values, int limit)
     {
-        checkArgument(values != null && values.size() > 0);
+        checkArgument(values.size() > 0);
```

Comment:
This is incorrect replacement

Reply:
This is deliberate. values== null causes a NullPointerException. values <= 0 causes an IllegalArgumentException